app/views/layouts/user_mailer.html.erb

```diff
@@ -0,0 +1,183 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
+<html
+	xmlns="http://www.w3.org/1999/xhtml">
```

Comment:
this looks like tabs not spaces ಠ\_ಠ, can we convert back to spaces please?

Reply:
Good catch, just fixed